rpm/get_version_string.sh

```diff
@@ -1,8 +1,19 @@
 #!/bin/bash
+
+# Copyright 2014 Telefonica Investigacion y Desarrollo, S.A.U
```

Comment:
2014 -> 2015

(Except if you want to consider the date of the original script, that -maybe, I'm not sure- was created in 2014).